### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DavidBrockway**
New Design Layout, more information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,60 +1,73 @@
...
+Steadfast2 is a project for backporting new Minecraft: Pocket Edition changes to older Pocketmine versions for better stability and performance, while retaining as many features from the new versions as possible. It's currently in production on Lifeboat Survival Games' main servers.
 
-#### Known bugs:
+## Known bugs
 
-Things you'll want to change on your plugins:
-   - Players don't fall out of the world naturally, you'll want to handle PlayerMoveEvent as needed to kill them
-   
+- Players don't fall out of the world naturally, you'll want to handle PlayerMoveEvent as needed to kill them.
+- Chunks won't load, you will want to go to server.properties and set auto-generate value as on.
```
Comment (vchibikov) [inline README.md]: This is not a bug))

**Comment by DavidBrockway** [comment]
Corrected

**Comment by DavidBrockway** [comment]
Can it be merged now>

**Comment by SuperMaXAleX** [comment]
Stop forcing and wait.

**Comment by DavidBrockway** [comment]
Sure

**Pull request merged**